includes/Story_Post_Type.php

```diff
@@ -301,6 +305,10 @@ public static function admin_enqueue_scripts( $hook ) {
 						'statuses' => '/wp/v2/statuses',
 						'fonts'    => '/amp/v1/fonts',
 					],
+					'metadata'         => [
+						'publisher'     => $publisher,
+						'publisherLogo' => $publisher_logo,
+					],
```

Comment:
Would the user expect this to update "live" in the editor if changing it in another tab/window? The above obv only updates on refresh.

Reply:
This rarely ever changes I think though

Reply:
Don't we need to update [this](https://github.com/google/web-stories-wp/blob/40cdc3e9f92b4569c91b229fec8bfe6e9f09c8e3/.storybook/stories/playground/index.js#L27) as well.